src/utils/graphql.js

```diff
@@ -66,6 +67,16 @@ function graphqlFieldToString(field: Object): FlowType {
     };
   }
 
+  if (graphqlType instanceof GraphQLList) {
+    const children = graphqlFieldToString(graphqlType.ofType);
+
+    return {
+      type: "array",
+      nullable,
+      children: children.type === "object" ? convertGraphqlObjectType(graphqlType.ofType) : null
```

Comment:
I don't know why `children.type === "object"` typechecks with flow either, it should really be `graphqlType.ofType instanceof GraphQLObjectType` as that ensures a `GraphQLObjectType` is being passed to `convertGraphqlObjectType`.